pallets/proxy-rmrk-core/src/rpc.rs

```diff
@@ -235,6 +261,7 @@ pub fn nft_resources<T: Config>(
 	Ok(resources)
 }
 
+/// Get the priority of a resource in an NFT.
```

Comment:
What is the 'priority of a resource' term means?

Reply:
It's a RMRK thing, not one of their main features, so I didn't see the point of transferring even more of their docs here :) It's better described in the transaction body [here](https://github.com/UniqueNetwork/unique-chain/blob/doc/rmrk/pallets/proxy-rmrk-core/src/lib.rs#L1150)